hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/FeatureProvider.java

```diff
@@ -85,7 +87,9 @@ public static void initFeatureSupport(
     resetInitOfFeatureSupport();
     String heatMapProviderCls = ozoneConfiguration.get(
         OZONE_RECON_HEATMAP_PROVIDER_KEY);
-    if (StringUtils.isEmpty(heatMapProviderCls)) {
+    boolean heatMapEnabled = ozoneConfiguration.getBoolean(
+        OZONE_RECON_HEATMAP_ENABLE_KEY, OZONE_RECON_HEATMAP_ENABLE_DEFAULT);
+    if (!heatMapEnabled || StringUtils.isEmpty(heatMapProviderCls)) {
```

Comment:
we can rename interface as featureDisableMap, to match behaviour

Reply:
Done.